runtime/parachains/src/disputes.rs

```diff
@@ -1038,74 +964,19 @@ impl<T: Config> Pallet<T> {
 			return StatementSetFilter::RemoveAll
 		}
 
-		// Apply spam slot changes. Bail early if too many occupied.
+		// Reject disputes containing less votes than needed for confirmation.
+		if (summary.state.validators_for.clone() | &summary.state.validators_against).count_ones() <=
```

Comment:
We could be a bit more lenient in fact. In case the dispute is local, we actually don't need confirmation to know that the dispute is no spam.

Reply:
Okay, just to be sure I understand. The reasoning is 'we have to look up the candidate to determine if it is local => if we can look it up it's not fake'?

Reply:
Reverted